starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,44 @@
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+
+h1, body, html, main {
+  margin: 0;
+}
+
+.applepieimage {
+  background-image: url(../apple-pie-starter-code/images/apple-pie.jpg);
```

Comment:
The correct path would be: 
`background-image: url("../images/apple-pie.jpg");`
so make sure you navigate properly inside the working folder and to put the path into quotes.